letsencrypt/README.rst

```diff
@@ -88,7 +89,21 @@ You'll also need a matching sudo configuration, like::
 
     your_odoo_user ALL = NOPASSWD: /usr/sbin/service nginx reload
 
-Further, if you force users to https, you'll need something like for nginx::
+The line above can be added to /etc/sudoers through the visudo command, or
+even better be put in a separate file in the /etc/sudoers.d directory. For
+instance in a file /etc/sudoers.d/letsencrypt. (Name should not end in ~ and
+should not contain a dot). This file should be owned by root and have
+authorities set by ``chown 0440 your_filename``.
```

Comment:
as you give a linux admin course here anyways, propose to edit this file with `visudo -f`, this also creates it with correct permissions